pandas/tests/io/formats/test_format.py

```diff
@@ -3219,6 +3231,67 @@ def test_period_tz(self):
             per = dt.to_period(freq="H")
         assert per.format()[0] == "2013-01-01 00:00"
 
+    @pytest.mark.parametrize(
+        "locale_str",
+        [
+            pytest.param(None, id=str(locale.getlocale())),
+            "it_IT.utf8",
+            "it_IT",  # Note: encoding will be 'ISO8859-1'
```

Comment:
Might just be better to set `it_IT.ISO8859-1`?

Reply:
I have tried it, but unfortunately it does not work on my machine (windows)